pkg/app/appdisc/discovery_manager.go

```diff
@@ -1,15 +1,14 @@
-package updatedisc
+package appdisc
 
 import (
 	"context"
 	"strconv"
 	"sync"
-	"time"
 
 	"github.com/skycoin/skywire/pkg/servicedisc"
 )
 
-// Updater updates the associated app discovery
+// Updater update the associated app discovery
```

Comment:
`updates` is the right word here so no need to change it to `update`

Reply:
Done